ci/ci_test.sh

```diff
@@ -7,6 +7,8 @@
 
 SELECTED_TEST_SUITE=$1
 
+export SPEC_OPTS="--order rand --format documentation --format CI::Reporter::RSpecFormatter"
```

Comment:
I don't understand, documentation is the default in ```test.rake``` but is also set here? is this done in any purpose I don't see?